README.md

```diff
@@ -108,9 +108,9 @@ Which would yield
 
 ```
 {
-    "prediction": [
-        {"z": 11525},
-        {"z": 3457}
+    [
```

Comment:
I prefer the previous format (mostly because it's smaller with a large number of predictions) but I could be convinced. 
Also I think the old model would still return results in the old format no?

Reply:
I agree the format described by the README.md is better but the real format is the one as edited (for v3 and v4 at least).
As it was out of scope I just updated the README.md to reflect current state.

Reply:
If it was a documentation mistake, then it's good you fixed it and I agree that it's out of scope to change the output format.